src/standard.ts

```diff
@@ -1,14 +1,20 @@
-import { keccak256 } from 'ethereum-cryptography/keccak';
 import { equalsBytes, hexToBytes } from 'ethereum-cryptography/utils';
-import { defaultAbiCoder } from '@ethersproject/abi';
 import { Bytes, compareBytes, hex } from './bytes';
 import { getProof, isValidMerkleTree, makeMerkleTree, processProof, renderMerkleTree, MultiProof, getMultiProof, processMultiProof } from './core';
 import { checkBounds } from './utils/check-bounds';
 import { throwError } from './utils/throw-error';
+import { standardLeafHash } from './utils/standard-leaf-hash';
 
-function standardLeafHash<T extends any[]>(value: T, types: string[]): Bytes {
-  return keccak256(keccak256(hexToBytes(defaultAbiCoder.encode(types, value))));
-}
+// MerkleTree building options
+export type StandardMerkleTreeOptions = Partial<{
+  sortLeaves: boolean;
```

Comment:
The user will see this in their IDE when setting the option:

```suggestion
  /** Enable or disable sorted leaves. Sorting is strongly recommended for multiproofs. */
  sortLeaves: boolean;
```